go/client/cmd_simplefs.go

```diff
@@ -69,6 +69,12 @@ func makeSimpleFSPath(g *libkb.GlobalContext, path string) keybase1.Path {
 
 	path = filepath.ToSlash(filepath.Clean(path))
 
+	// Certain users seem to want to use SimpleFS on their
+	// mounted KBFS
+	if strings.HasPrefix(path, mountDir) {
+		return keybase1.NewPathWithKbfs(path[len(mountDir):])
+	}
```

Comment:
This seems duplicated from line 55 above. Is that intended?

Also it looks like this is relying on `mountDir` not having a trailing slash. Is that something we can rely on?